code/game/objects/items/tools/multitool.dm

```diff
@@ -110,22 +110,34 @@
 	name = "command multitool"
 	desc = "Used for pulsing wires to test which to cut. Not recommended by the Captain."
 	icon_state = "multitool_command"
+	item_state = "multitool_command"
 	belt_icon = "multitool_command"
 	toolspeed = 0.95 //command those wires / that fireaxe cabinet!
+	var/list/victims = list()
 
 /obj/item/multitool/command/suicide_act(mob/living/user)
 	user.visible_message("<span class='suicide'>[user] is attempting to command the command multitool! It looks like [user.p_theyre()] trying to commit suicide!</span>")
     //basically just cleaned up and copied from the medical wrench code
 	if(!user)
 		return
 
-	user.Stun(10 SECONDS)
-	playsound(loc, 'sound/effects/supermatter.ogg', 50, TRUE, -1)
+	user.Immobilize(10 SECONDS)
 	sleep(20)
-
 	add_fingerprint(user)
-	desc += " Its screen displays the text \"[user.name]: executed for mutiny.\""
 
+	var/base_desc = "Used for pulsing wires to test which to cut. Not recommended by the Captain."
+	switch(length(victims))
+		if(0)
+			desc += " Its screen displays the text \"[user.name]: executed for mutiny.\""
+		if(1)
+			desc = base_desc + " Its screen displays the text \"[victims[1]] & [user.name]: executed for mutiny.\""
+		else
+			desc = base_desc + " Its screen displays the text \""
+			for(var/victim_name in victims)
```

Comment:
You can probably also use the `english_list` proc to make the code a little cleaner here, it's useful for formatting things like this.